tests/unit/election-lookup.js

```diff
@@ -0,0 +1,87 @@
+'use strict'
+
+/* global require, window, describe, before, beforeEach, after, afterEach, it */
+
+var chai = require('chai');
+var sinon = require('sinon');
+var sinonChai = require('sinon-chai');
+var expect = chai.expect;
+chai.use(sinonChai);
+
+var $ = require('jquery');
+var _ = require('underscore');
+
+var ElectionLookup = require('../../static/js/pages/election-lookup').ElectionLookup;
+
+_.extend(window, {
```

Comment:
This should eventually be moved into a location and then all tests will import it.